config/admin-functions/create-machine-cert.sh

```diff
@@ -112,6 +113,15 @@ function create_machine_cert_signing_request() {
         -out "${cert_signing_request_path}"
 }
 
+# Permissions helpers
+
+# Applies permissions to match the permissions of other non-executable files in VX_CONFIG_ROOT
+function match_vx_config_non_executable_file_permissions() {
+    local file_path="${1}"
+    chown vx-admin:vx-group "${file_path}"
+    chmod u=rw,g=r,o= "${file_path}"
+}
```

Comment:
Used these permissions as my reference:

https://github.com/votingworks/vxsuite-complete-system/blob/8c0529350a7b2cbd36ed95be543bdb4344705e52/setup-machine.sh#L326-L330

The `X` isn't needed since I know the files I'm working with aren't executable. (Capital `X` is smart and will only apply executable permissions if the file is already executable in some way, but I didn't want to be misleading here.)